azure/image/Makefile

```diff
@@ -8,22 +8,14 @@ include ../../podvm/Makefile.inc
 
 image: $(IMAGE_FILE)
 
-$(IMAGE_FILE): $(BINARIES) $(FILES)
+$(IMAGE_FILE): $(BINARIES) $(FILES) packer-build
+
+# When doing a packer build we already have dependencies built. We don't need to build things like kata-agent, etc.
+image-build-in-docker: packer-build
```

Comment:
if the components were already built and extracted under files `make image` should also skip building of existing targets(files) IIUC no?

Reply:
Yeah it should, but not sure why it would always try to build with just `make image`.

Reply:
> Yeah it should, but not sure why it would always try to build with just make image.

`make` always builds the binaries, because their dependency is their source code, which is unconditionally fetch from GitHub.

This unmerged commit is fixing a similar issue.  https://github.com/confidential-containers/cloud-api-adaptor/pull/800/commits/93c0a00c011114619d9900d5b97d24d6c5e33135

I think we can address this issue in a separated PR.

Reply:
I think this should not happen with `BINARIES=`
see: 0d6eda047e81eadbec3145407d2e55a6244f83cc
cc @bpradipt